### PR TITLE
Add content/pt-br/docs/setup/production-environment/tools/kops.md

### PR DESCRIPTION
**Pull request by juliafmorgado**
Added pt-br translation for Kops page.

### CODE REVIEW

File: content/pt-br/docs/setup/production-environment/tools/kops.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+### (4/5) Crie sua configuração de cluster
+
+Execute `kops create cluster` para criar sua configuração de cluster:
+
+`kops create cluster --zones=us-east-1c useast1.dev.example.com`
+
+kops criará a configuração para seu cluster. Observe que ele _apenas_ cria a configuração, na verdade não cria os recursos de nuvem - você fará isso na próxima etapa com um arquivo `kops update cluster`.  Isso lhe dá a oportunidade de revisar a configuração ou alterá-la.
+
+Ele imprime comandos que você pode usar para explorar mais:
```
Comment (edsoncelio) [inline content/pt-br/docs/setup/production-environment/tools/kops.md:166]: "imprime" -> "exibe"

**Review by edsoncelio — COMMENTED** [review]
Hi @juliafmorgado, first of all - thank you for your contribution!
I left a few suggestions/fixes.

Just another thing - could you remove the change in the file `content/en/docs/setup/production-environment/tools/kubespray.md`? As we talked you will open another PR to fix that in the original content.
Thanks!

File: content/pt-br/docs/setup/production-environment/tools/kops.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+Este início rápido mostra como instalar facilmente um cluster Kubernetes na AWS usando uma ferramenta chamada [`kOps`](https://github.com/kubernetes/kops). 
+
+`kOps` é um sistema de provisionamento automatizado:
+
+* Instalação totalmente automatizada
+* Usa DNS para identificar clusters
+* Auto-recuperação: tudo é executado em grupos de Auto-Scaling
+* Suporte de vários sistemas operacionais (Amazon Linux, Debian, Flatcar, RHEL, Rocky e Ubuntu) - veja as [imagens.md](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md)
+* Suporte de alta disponibilidade - consulte o [high_availability.md](https://github.com/kubernetes/kops/blob/master/docs/operations/high_availability.md)
+* Pode provisionar diretamente ou gerar manifestos do terraform - veja o [terraform.md](https://github.com/kubernetes/kops/blob/master/docs/terraform.md)
```
[the remark this answers is not in the document]
Comment (juliafmorgado) [reply content/pt-br/docs/setup/production-environment/tools/kops.md:18]: Modificado.

File: content/pt-br/docs/setup/production-environment/tools/kops.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+## {{% heading "prerequisites" %}}
+
+* Você deve ter o [kubectl](/docs/tasks/tools/) instalado.
+
+* Você deve [instalar](https://github.com/kubernetes/kops#installing) `kops` em uma arquitetura de dispositivo de 64 bits (AMD64 e Intel 64).
+
+* Você deve ter uma [conta da AWS](https://docs.aws.amazon.com/polly/latest/dg/setting-up.html), gerar [chaves do IAM](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) e [configurá-las](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#cli-quick-configuration). O usuário do IAM precisará de [permissoões adequadas](https://github.com/kubernetes/kops/blob/master/docs/getting_started/aws.md#setup-iam-user).
```
[the remark this answers is not in the document]
Comment (juliafmorgado) [reply content/pt-br/docs/setup/production-environment/tools/kops.md:26]: Modificado.

File: content/pt-br/docs/setup/production-environment/tools/kops.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+### (1/5) Instalar kops
+
+#### Instalação
+
+Faça o download do kops na [página de lançamentos](https://github.com/kubernetes/kops/releases) (também é conveniente compilar a partir da fonte):
+
+{{< tabs name="instalação_kops" >}}
+{{% tab name="macOS" %}}
+
+aixe a versão mais recente com o comando:
```
[the remark this answers is not in the document]
Comment (juliafmorgado) [reply content/pt-br/docs/setup/production-environment/tools/kops.md:41]: Editado.

File: content/pt-br/docs/setup/production-environment/tools/kops.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)
+```
+
+Por exemplo, para baixar kops versão v1.20.0 digite:
+
+```shell
+curl -LO https://github.com/kubernetes/kops/releases/download/v1.20.0/kops-darwin-amd64
+```
+
+Torne o kops executável binário.
```
[the remark this answers is not in the document]
Comment (juliafmorgado) [reply content/pt-br/docs/setup/production-environment/tools/kops.md:59]: This was a hard one, thanks for the help!

File: content/pt-br/docs/setup/production-environment/tools/kops.md (hunk 1 above)
Comment (juliafmorgado) [reply content/pt-br/docs/setup/production-environment/tools/kops.md:166]: Alterado.

File: content/pt-br/docs/setup/production-environment/tools/kops.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+`kops create cluster --zones=us-east-1c useast1.dev.example.com`
+
+kops criará a configuração para seu cluster. Observe que ele _apenas_ cria a configuração, na verdade não cria os recursos de nuvem - você fará isso na próxima etapa com um arquivo `kops update cluster`.  Isso lhe dá a oportunidade de revisar a configuração ou alterá-la.
+
+Ele imprime comandos que você pode usar para explorar mais:
+
+* Liste seus clusters com: `kops get cluster`
+* Edite este cluster com: `kops edit cluster useast1.dev.example.com`
+* Edite seu grupo de instâncias de nós: `kops edit ig --name=useast1.dev.example.com nodes`
+* Edite seu grupo de instâncias mestre: `kops edit ig --name=useast1.dev.example.com master-us-east-1c`
```
[the remark this answers is not in the document]
Comment (juliafmorgado) [reply content/pt-br/docs/setup/production-environment/tools/kops.md:171]: Alterado.

File: content/pt-br/docs/setup/production-environment/tools/kops.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+
+<!-- steps -->
+
+## Como criar um cluster
+
+### (1/5) Instalar kops
+
+#### Instalação
+
+Faça o download do kops na [página de lançamentos](https://github.com/kubernetes/kops/releases) (também é conveniente compilar a partir da fonte):
```
[the remark this answers is not in the document]
Comment (juliafmorgado) [reply content/pt-br/docs/setup/production-environment/tools/kops.md:36]: Yes much better!

**Pull request closed without merging**